Add mapping between Steps and SVG elements.

dhavalpzala opened the pull request:
-Add remove steps functionality
-Implement rxjs event messages
-Change SVG element selection using tab key.
-Change step selection according to SVG element selection.

dhavalpzala committed:
Remove Steps and Event Messages
-add remove steps functionality
-implement rxjs event messages

dhavalpzala committed:
Add binding between Steps and SVG elements.
-Change SVG element selection using tab key.
-Change step selection according to SVG element selection.

hashd commented on src/dvu/core/step.ts:
```diff
@@ -3,23 +3,31 @@ import {Scope} from './scope'
 import {ValueType} from './data/data_definition'
 
 export class Step {
+  private element: Element
```
Can we avoid having element mapped against a step? Maybe we could have a https://en.wikipedia.org/wiki/Bidirectional_map which maintains this information in the interface itself.

dhavalpzala replied:
Where should we add that bidirectional mapping?

dhavalpzala committed:
Select SVG element using click.

dhavalpzala committed:
Add uuid property in Step
-add unique id for Step.
-remove element property from Step.
-create stepElementMap to track step and element mapping.

hashd commented:
Merged with https://github.com/hashd/papyrus/commit/5b73842ee092e51378db6fbbe3c6854912fbe244
